package.json

```diff
@@ -15,7 +15,7 @@
     "commit": "cz",
     "start:prod": "npm run prisma:deploy && NODE_ENV=production node --trace-warnings ./dist/server.js",
     "type-check": "tsc --noEmit",
-    "lint": "eslint ./src",
+    "lint": "eslint ./src --resolve-plugins-relative-to .",
```

Comment:
why this change?

Reply:
This was to fix an issue I was having with eslint saying it was missing a bunch of dependencies. If I'm being honest I could probably revert back to the older versions and everything should work 🤞.

I'll give that a go.